src/domain/collection/eventList/EventList.tsx

```diff
@@ -39,6 +39,7 @@ const EventList: React.FC<Props> = ({ collection }) => {
       params: searchParams,
       sortOrder: EVENT_SORT_OPTIONS.END_TIME,
       superEventType: ['umbrella', 'none'],
+      eventType: 'event',
```

Comment:
This will need to support courses also, but that will be done in my pull request #280. Or did I miss course implementation?